contribs/github-bot/internal/config/config.go

```diff
@@ -54,6 +54,19 @@ func Config(gh *client.GitHub) ([]AutomaticCheck, []ManualCheck) {
 			If:          c.Label("don't merge"),
 			Then:        r.Never(),
 		},
+		{
+			Description: "Pending initial approval by a review team member (and label matches review triage state)",
+			If:          c.Not(c.AuthorInTeam(gh, "tech-staff")),
+			Then: r.
+				If(r.Or(r.ReviewByOrgMembers(gh, 1), r.Draft())).
+				// Either there was a first approval from a member, and we
+				// assert that the label for triage-pending is removed...
+				Then(r.Not(r.Label(gh, "review/triage-pending", r.LabelRemove))).
+				// Or there was not, and we apply the triage pending label.
+				// The requirement should always fail, to mark the PR is not
+				// ready to be merged.
+				Else(r.And(r.Label(gh, "review/triage-pending", r.LabelApply), r.Never())),
+		},
```

Comment:
After rereading, I wonder if we should separate _requirements_ and _actions_ (e.g., adding/removing a label). 
Because in fact, these are not requirements to determine whether the CI fails or not, the only requirement here is `ReviewByOrgMembers(1)`. 

Maybe by adding a third type of config like this one?
```
type AutomaticAction struct {
	Description  string
	If           c.Condition
	Then         a.Action // Add/Remove label, Assign/Unassign user, etc.
}
```

But I don't think there is any ideal solution anyway.
Also it would require quite a bit of extra work to be implemented, and I think your solution works even though it makes readability a bit difficult. 

I'm totally okay to merge it as it is and already test how it turns out. 👍

Reply:
I think the ramifications are complex, and this is a larger refactor (think, aside from `label`, that we perform "actions" also for assignee and reviewer). I think you can try tackling this in another PR, also because I'm not sure how to make the current "reviewer" requirement work with an `Action`, so I'd say to merge this today :)